react_client/src/api/localStateAPI.jsx

```diff
@@ -28,7 +28,7 @@ export function getLocalState(
 // POST state changes
 export function postLocalState(endpoint, stateData1, stateData2) {
   axios
-    .post(`http://localhost:5523/${endpoint}`, {
+    .post(`/api/${endpoint}`, {
```

Comment:
because we all using /api in the server, this needs to get tht update. not the leading slah, this is enabling the api rewrite to work!